### ISSUE TITLE
BOOK_AND_QUILL still referenced in uSkyBlock for 1.13

### ISSUE THREAD

**Issue by rlf** [rlf/uSkyBlock#1094]
See https://www.spigotmc.org/threads/uskyblock.38300/page-134#post-3141493

**Comment by rlf** [comment]
`mvn clean install` provided you have maven installed, if you want the i18n files as well (.po files), you need to add the i18n profile (and possibly tweak stuff for where the Gettext binaries are located).

**Comment by SullyTheUnusual** [comment]
Main branch builds fine, but tried building 2.7.6-RC5 with maven, and got this error:  https://hastebin.com/epalugohih.cs

**Comment by MCRakaan** [comment]
So it is known what is the problem, and easy fixable.
Why the long wait for a compiled version?

**Comment by Jamie-Vangeysel** [comment]
Getting the same error mentioned by @SullyTheUnusual 
<img width="1457" alt="schermafbeelding 2018-09-09 om 00 45 09" src="https://user-images.githubusercontent.com/32323785/45259294-b4ec8d00-b3c9-11e8-9109-bd5887369d06.png">

**Comment by MCRakaan** [comment]
Oh wow... i actuly managed to do it. *snif* my first git compile
Linux maven 3.5
```
git clone https://github.com/rlf/uSkyBlock.git
cd uSkyBlock
mvn clean install
cd uSkyblock-Plugin/target
```
The uSkyblock.jar is the one you want.

**Comment by SullyTheUnusual** [comment]
Yeah, like I said above, I did the same thing.  Maven says it's compiling version 2.7.4, but now that I check, the challenges command works fine.  I guess someone forgot to update the version number on the main branch?
